src/components/templates/AnimateMap/hooks/useRecentLocationsUsers.ts

```diff
@@ -46,10 +46,9 @@ export const useRecentLocationsUsers = (
               worldUserLocationsById[user.id];
 
             return (
-              userLocation.lastSeenIn?.[venueName] &&
-              normalizeTimestampToMilliseconds(
-                userLocation.lastSeenIn[venueName]
-              ) > lastSeenThreshold
+              userLocation.lastSeenIn === venueName &&
```

Comment:
Doesn't this need to be modified like https://github.com/sparkletown/sparkle/pull/2202/files#diff-69b65f1189b0befb788170375aded3d463c6398d8804305bab5072c7624a5e93R44 ?

Reply:
Good catch. Done in https://github.com/sparkletown/sparkle/pull/2202/commits/c10b9c5940ef0d49677132d8549ebe515c2217e3